specification/monitor/resource-manager/Microsoft.Insights/stable/2021-05-01/metricDefinitions_API.json

```diff
@@ -0,0 +1,457 @@
+        },
+        "namespace": {
+          "type": "string",
+          "description": "the namespace the metric belongs to."
+        },
+        "name": {
+          "$ref": "#/definitions/LocalizableString",
+          "description": "the name and the display name of the metric, i.e. it is a localizable string."
+        },
+        "displayDescription": {
+          "type": "string",
+          "description": "Detailed description of this metric."
+        },
+        "category": {
+          "type": "string",
+          "description": "Custom category name for this metric."
+        },
+        "metricClass": {
+          "$ref": "#/definitions/MetricClass",
+          "description": "The class of the metric."
+        },
+        "unit": {
+          "$ref": "#/definitions/MetricUnit",
+          "description": "the unit of the metric."
+        },
+        "primaryAggregationType": {
+          "$ref": "#/definitions/MetricAggregationType",
+          "description": "the primary aggregation type value defining how to use the values for display."
+        },
+        "supportedAggregationTypes": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/MetricAggregationType"
+          },
+          "description": "the collection of what aggregation types are supported."
+        },
+        "metricAvailabilities": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/MetricAvailability"
+          },
+          "description": "the collection of what aggregation intervals are available to be queried."
+        },
+        "id": {
+          "type": "string",
+          "description": "the resource identifier of the metric definition."
+        },
+        "dimensions": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/LocalizableString"
+          },
+          "description": "the name and the display name of the dimension, i.e. it is a localizable string."
+        }
+      }
+    },
+    "SubscriptionScopeMetricDefinitionCollection": {
+      "type": "object",
+      "description": "Represents collection of metric definitions.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SubscriptionScopeMetricDefinition"
+          },
+          "description": "The values for the metric definitions."
+        }
+      },
+      "required": [
+        "value"
+      ]
+    },
+    "ErrorContract": {
+      "title": "Error Response",
+      "description": "Common error response for all Azure Resource Manager APIs to return error details for failed operations. (This also follows the OData error response format.)",
+      "type": "object",
+      "properties": {
+        "error": {
+          "description": "The error object.",
+          "$ref": "#/definitions/ErrorResponse"
+        }
+      }
+    },
+    "ErrorResponse": {
```

Comment:
Same case as `ErrorContract`.

Reply:
Created a commonMonitoringTypes.json file and moved it there.